src/Sentry.Bindings.Cocoa/Sentry.Bindings.Cocoa.csproj

```diff
@@ -113,6 +113,15 @@
     <Exec Condition="'$(GITHUB_ACTIONS)' == 'true'" Command="pwsh ../../scripts/dirty-check.ps1 -PathToCheck $(MSBuildThisFileDirectory)" />
   </Target>
 
+  <!-- Sanitize the xcframework before NativeReference resolution and packing -->
+  <Target Name="SanitizeSentryCocoaFramework"
+          AfterTargets="_GenerateSentryCocoaBindings"
+          BeforeTargets="ResolveNativeReferences"
+          Condition="$([MSBuild]::IsOSPlatform('OSX')) and Exists('$(SentryCocoaFramework)')">
+    <Message Importance="High" Text="Sanitizing $(SentryCocoaFramework): removing dSYMs, Headers, Modules, PrivateHeaders (including symlinks)." />
+    <Exec Command="bash -c 'find &quot;$(SentryCocoaFramework)&quot; -depth \( -name dSYMs -o -name Headers -o -name Modules -o -name PrivateHeaders \) -exec echo Removing {} \; -exec rm -rf {} + 2&gt;/dev/null || true'" />
```

Comment:
I think this executes for multiple target frameworks in parallel. Might be the reason you had to ignore errors because the two commands race and conflict?

What if this was already done at the end of `_SetupCocoaSDK`, which is guarded to run only once? If we need to ignore errors regardless, we could get away without bash with `<Exec IgnoreExitCode="true" ... />`

Reply:
P.S. `rm` has verbose mode (`-v`) with quite similar output